pyiron_base/project/maintenance.py

```diff
@@ -60,9 +60,13 @@ def get_repository_status():
             module = importlib.import_module(name)
             try:
                 repo = Repo(os.path.dirname(os.path.dirname(module.__file__)))
-                hash_ = repo.head.reference.commit.hexsha
             except InvalidGitRepositoryError:
                 hash_ = "Not a repo"
+            else:
+                try:
+                    hash_ = repo.head.reference.commit.hexsha
+                except ValueError:
```

Comment:
For detached HEAD, a type errror is raised

```suggestion
                except (ValueError, TypeError):
```